packages/xpath/src/functions/javarosa/index.ts

```diff
@@ -1,5 +1,5 @@
 import { JAVAROSA_NAMESPACE_URI } from '@getodk/common/constants/xmlns.ts';
 import { FunctionLibrary } from '../../evaluator/functions/FunctionLibrary';
-import * as string from './string.ts';
+import * as nodeSet from './node-set.ts';
 
-export const jr = new FunctionLibrary(JAVAROSA_NAMESPACE_URI, [...Object.values(string)]);
+export const jr = new FunctionLibrary(JAVAROSA_NAMESPACE_URI, [...Object.values(nodeSet)]);
```

Comment:
Fact: Currently, there can only be one function for `itext`. I tried leaving string and node-set, but it just keeps one. Knowing that `itext` will always return one data type from the XPath layer could be a good thing.